### ISSUE TITLE
[BUG] Upload of none-existant image throws exception in backend

### ISSUE THREAD

**Issue by madc** [bolt/bolt#6252]
Client uses Safari as a Browser to upload images via the `Select File` button in `Uploaded files`. The files seem to located on some remote (SMB?) share. Now when he clicks the button, the system dialog shows up as expected, but does show the filestructure from some time ago (no recent changes are shown). As the system dialog shows an outdated file-list of the server, the client managed to upload a 0kb file, simply by selecting a file thats not existing anymore..

Now this is properly an odd safari issue, nothing that should concern us here, but..

The problem is, once this 0kb image file is on the server, it is impossible to access the folder, in which it is in, via the backend.

![bolt_image_error](https://cloud.githubusercontent.com/assets/343392/21929613/eb106064-d991-11e6-863d-7f56f1d91e33.png)

Details
-------

 - Relevant Bolt Version: 3.2.4
 - Install type: GitHub checkout
 - PHP version: [ 5.5 | 5.6 | 7.0 ]
 - Used web server: Apache
 - For UX/UI issues: Safari OsX

Reproduction
------------

It should be enough to upload an empty file with a image-ext (.jpg in our case) and try to navigate there in the backend.

**Comment by GawainLynch** [comment]
The specific part of the trace:
```
1. in [snip]/vendor/bolt/filesystem/src/Handler/Image/Info.php line 92
2. at Info::createFromString(' ') in [snip]/vendor/bolt/filesystem/src/Filesystem.php line 664
3. at Filesystem->getImageInfo('agriculture-cereals-field-621.jpg') in [snip]/vendor/bolt/filesystem/src/Handler/Image.php line 24
4. at Image->getInfo() in [snip]/vendor/bolt/filesystem/src/Handler/Image.php line 40
```

**Comment by GawainLynch** [comment]
@CarsonF ideas on approach here?

**Comment by CarsonF** [comment]
@GawainLynch Does the same thing happen on 3.3? Again all rewritten there...

**Comment by GawainLynch** [comment]
It does not happen on 3.3. Is this hotifx-able, or pressure on 3.3 release?

**Comment by CarsonF** [comment]
I would rather just pressure on 3.3 release. I would be worried about unintended results of "fixing" it here. The fact that there's cached data is curious too. I don't even remember what Bolt is capable of prior to 3.3 in regards to remote FS. And if they are using remote filesystems, 3.3 should help out a lot.

Still I'm not sure what I would've changed in 3.3 to make this not a problem.

**Comment by bobdenotter** [comment]
marking as wontfix, if it's solved in 3.3
